components/expenses/Order.js

```diff
@@ -26,7 +30,7 @@ class Order extends React.Component {
     includeHostedCollectives: PropTypes.bool,
     LoggedInUser: PropTypes.object,
     intl: PropTypes.object.isRequired,
-    onClickCancel: PropTypes.func,
+    markPendingOrderAsExpired: PropTypes.func,
```

Comment:
If prop is mandatory then let's make it explicit

```suggestion
    markPendingOrderAsExpired: PropTypes.func.isRequired,
```